front/src/routes/scene/edit-scene/actions/DeviceSetValue.jsx

```diff
@@ -136,6 +192,22 @@ class DeviceSetValue extends Component {
               <Text id="global.requiredField" />
             </span>
           </label>
+          <div className="nav-tabs" style={{ display: 'flex', margin: 0 }}>
+            <span
+              class={`nav-link ${this.state.computed ? '' : 'active'}`}
+              style={'width: 50%'}
+              onClick={() => this.setState({ computed: false })}
```

Comment:
We try to avoid this pattern in the front because it creates lots of function on each render

Prefer a fixed function that never changes :)